ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/ipv4_acls.schema.yml

```
@@ -0,0 +1,45 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ipv4_acls:
+    type: list
+    primary_key: name
+    description: |-
+      IPv4 Extended Access-lists supporting substition on certain fields.
```

```suggestion
      IPv4 Extended Access-lists supporting substitution on certain fields.
```
Same in other places

---

Not sure why my brain has made the same spelling mistake all over the place...